src/tabs/tab-header-bar.scss

```diff
@@ -113,11 +115,6 @@ $label-horizontal-spacing: awsui.$space-xs;
   display: flex;
   align-items: stretch;
 
-  &,
-  & > button {
-    background-color: transparent;
```

Comment:
Removing `transparent` because it overrides custom styles for invalid or disabled. Removal caused no visual regressions. Keeping this property would prevent custom styles from taking effect.